src/main/java/bio/terra/common/stairway/StairwayJdbcProperties.java

```diff
@@ -0,0 +1,7 @@
+package bio.terra.common.stairway;
+
+import bio.terra.common.db.JdbcProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "terra.common.stairway.db")
```

Comment:
Here I'm attempting to define one properties class identical to another but with a different prefix. I guess it _could_ work. We need to do the same thing with `BufferJdbcProperties`.